src/main/kotlin/io/stud/forest/book_a_cinema_ticket_app/application/dto/RepertoireDto.kt

```diff
@@ -0,0 +1,43 @@
+package io.stud.forest.book_a_cinema_ticket_app.application.dto
+
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
+import io.stud.forest.book_a_cinema_ticket_app.domain.entities.Screening
+import io.stud.forest.book_a_cinema_ticket_app.domain.repositories.Id
+import java.time.LocalDateTime
+
+@JsonPropertyOrder("movieTitle", "timetable")
+data class RepertoireDto(
+        val movieTitle: String,
+        val timetable: List<TimeTableItem>,
+) {
+    companion object {
+        fun listOf(screenings: List<Screening>): List<RepertoireDto> =
+                screenings
+                        .groupBy { it.movie.title }
+                        .map(sortEachGroupByStartTime)
+                        .map(mapGroupToDto)
+                        .sortedBy { it.movieTitle }
```

Comment:
Jak to zadziała z polskimi znakami w tytule? Tudzież z wielkością liter?

Reply:
Porówna znaki po kodzie ascii, więc nie tak jak miało być. 
Trzeba by użyć sortedWith i zaproponować własny Comparator.

Reply:
Jest jeszcze coś takiego jak collatory w Javie :)